src/index.ts

```diff
@@ -484,6 +487,13 @@ function startWorkerThread<R, T extends AnyAsyncFn<R>>(
 
   const useEval = isTs ? !tsUseEsm : !jsUseEsm && useGlobals
 
+  const sharedBuffer = new SharedArrayBuffer(bufferSize)
+  const sharedBufferView = new Int32Array(sharedBuffer)
+
+  // SharedArrayBuffer is faster to pass a response back, but has a limited size.
+  // We use this feature only if we can automatically resize the SharedArrayBuffer.
+  const useBuffer = sharedBuffer.growable
```

Comment:
If we always pass `sharedBuffer`, then we can already determine whether  to use it via property directly. I mean this variable could be removed, right? Or we should check via prototype before creating a new instance, so that we can pass different worker data accordingly.

Reply:
We need the buffer anyway to use the `Atomics.wait` so it doesn't cost more to check if it is growable